Changelog.md

```diff
@@ -1,5 +1,36 @@
 ## Ignition Msgs 8.x
 
+### Ignition Msgs 8.3.0 (2022-03-24)
+1. backport NavSat message for bridge compatibility
+    * [Pull request #231](https://github.com/ignitionrobotics/ign-msgs/pull/231)
```

Comment:
this entry can also be removed since it doesn't add any new changes to ign-msgs8

Reply:
Done